### DISCUSSION TITLE
阿里云盘找不到资源file_id，文件跟存在的

### DISCUSSION THREAD

**Discussion by hjs523** [AlistGo/alist#4483] [Q&A]
failed get objs: failed to list objs: NotFound.FileId:The resource file_id cannot be found. file_id:61d00fc108858617993f4974b58225d0277ba12d

**Comment by m1570908333** [comment]
> failed get objs: failed to list objs: NotFound.FileId:The resource file_id cannot be found. file_id:61d00fc108858617993f4974b58225d0277ba12d

请问你的解决了吗？

**Reply by hjs523** [reply]
没有呢

**Comment by linxuankent** [comment]
failed get objs: failed to list objs: NotFound.FileId:The resource file_id cannot be found
我也是

**Comment by guoweiyi** [comment]
failed get objs: failed to list objs: NotFound.FileId:The resource file_id cannot be found. 我也一样 Oauth令牌链接也已经换了 https://api.xhofe.top/alist/ali_open/token

**Comment by jhonchu5** [comment]
fs/list: NotFound.FileId:The resource file_id cannot be found.  今天开始阿里云open的驱动就一直报这个错误
<img width="1246" alt="image" src="https://github.com/alist-org/alist/assets/131732278/a9f26776-24de-431e-a955-c14e5ecdef70">

**Comment by xhofe** [comment]
alist 也只是拿现有的file_id去请求阿里云盘的api，api返回了此file_id不存在，我从来没有碰到过这个问题故无法复现。
你可以尝试：
- 强制刷新（右下角更多-刷新）上级目录
- 去阿里云盘官网检查是否存在此file_id，即进入：https://www.aliyundrive.com/drive/folder/file_id 替换file_id为实际报错的

**Reply by jhonchu5** [reply]
非常感谢，已经好了，出去旅游了1个礼拜，它自己好了，有可能那天阿里云抽风了

**Reply by Hyman00** [reply]
目前使用的最新的AList，在挂载阿里云盘分享资源时，也报同样错误：failed get objs: failed to list objs: The resource file_id cannot be found. not belong share

我看给的这个尝试url是进入到备份盘，我也在备份盘里创建了一个目录，并将其fileid更新到对应的存储设置里；发现还是报这个错误

不知道是哪个地方配置有问题？

**Reply by sxl47** [reply]
> alist 也只是拿现有的file_id去请求阿里云盘的api，api返回了此file_id不存在，我从来没有碰到过这个问题故无法复现。 你可以尝试：
> 
> * 强制刷新（右下角更多-刷新）上级目录
> * 去阿里云盘官网检查是否存在此file_id，即进入：https://www.aliyundrive.com/drive/folder/file_id 替换file_id为实际报错的

现在还是报错，这个地址已经变成了呀：
https://www.aliyundrive.com/drive/file/all/66a61xxxxxxxxxxxxxxxxxxxxxxxxx

**Comment by lzgbzat** [comment]
我今天配置的时候也是这个问题，自己创建的文件用后面的file_id都会出现这类错误，解决方法：在备份文件的文件夹下创建文件夹就是可以正常使用的

**Reply by anwen-anyi** [reply]
> 我今天配置的时候也是这个问题，自己创建的文件用后面的file_id都会出现这类错误，解决方法：在备份文件的文件夹下创建文件夹就是可以正常使用的

- 检查挂载类型是否正确

     - 如果文件来自阿里云盘非 备份文件 这个文件夹之外的 挂载类型选择 `资源库`
     - 如果来自阿里云盘 备份文件 文件夹内的，挂载类型 选择`备份盘`

**Reply by ysy** [reply]
我今天也遇到同样的问题，按你的方法在备份文件的文件夹下创建就可以正常使用了。谢谢！
